templates/events/index.html

```diff
@@ -0,0 +1,56 @@
+{% extends 'layout.html' %}
+
+{% load static %}
+
+{% block title %} Jagrati | Events {% endblock %}
+
+{% block style %}
+<style>
+    body {
+        background-color: #f7f3e9;
+    }
+    .card-img-top {
+    width: 100%;
+    height: 20vw;
+    object-fit: cover;
+    }
```

Comment:
The thumbnail gets distorted on the small screens.

![image](https://user-images.githubusercontent.com/39924567/119522352-82616900-bd99-11eb-8888-1d6997fd2826.png)

Reply:
will this do? it leaves white space at the extreme ends when screen size is reduced
![image](https://user-images.githubusercontent.com/55823965/119705381-4fd96e00-be76-11eb-81f4-ad672df7e64e.png)

Reply:
No, this doesn't look good. Can't you increase the height of the card-img for small screens?

Reply:
> No, this doesn't look good. Can't you increase the height of the card-img for small screens?

hi @garg3133 i made the necessary changes please review them